modules/nextflow/build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
     api "com.beust:jcommander:1.35"
     api("com.esotericsoftware.kryo:kryo:2.24.0") { exclude group: 'com.esotericsoftware.minlog', module: 'minlog' }
     api('org.iq80.leveldb:leveldb:0.12')
-    api('org.eclipse.jgit:org.eclipse.jgit:5.2.1.201812262042-r')
+    api('org.eclipse.jgit:org.eclipse.jgit:6.1.0.202203080745-r')
```

Comment:
oh no. new jgit landed here 😮‍💨

Reply:
sorry, I understood this change solves the issue with the pipeline, but I'll rollback